source/community/consortium/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: IIIF Consortium
+id: consortium
+layout: spec
+---
+
+## The IIIF Consortium
+
+The IIIF Consortium was formed in [June 2015][news] to provide steering and sustainability to the IIIF community by signing a [Memorandum of Understanding][mou] in Oxford, England.  The Consortium will provide ongoing oversight for the development of IIIF technologies and managing the growth of the community.
+
+## Staff
+
+The Consortium is currently in the process of hiring its first full-time staff member, and manages its finances through the [Council on Library and Information Resources][clir]
+
+## Members
+
+<ul>
+{% for i in site.data.institutions %}
+    {% if i.iiifc %}
+  <li>
+      {% if i.uri %}<a href="{{ i.uri }}">{% endif %}
+        {{ i.name }}
+      {% if i.uri %}</a>{% endif %}
+  </li>
+    {% endif %}
+{% endfor %}
+</ul>
+
+
+
+
+[mou]: mou/
```

Comment:
Relative links aren't supposed to work in Jekyll 3, but it seems like we're getting lucky. I'd change this to `/community/consortium/mou/`; we should already have absolute paths everywhere else (I did this as part of the upgrade a couple of months back).